### DISCUSSION TITLE
安装windows 3.26 版后，在浏览器中打开显示空白

### DISCUSSION THREAD

**Discussion by 455511dcvv** [AlistGo/alist#5079] [Q&A]
版本3.26，在浏览器中打开显示空白,测试http://127.0.0.1/ping 显示pong，浏览器换了谷歌，Microsoft Edge 手机都测试了，都是一样不显示内容，求告知

**Comment by xhofe** [comment]
打开开发者工具看一下报什么错

**Reply by 455511dcvv** [reply]
不是很会有具体操作吗，我把程序部署到win10系统下就能正常显示，win11打开就是空白

**Comment by anwen-anyi** [comment]
按键F12打开发者工具调试看看是不是这个错误？
![G1)TV`{1PU`QQ~1P}SX48V1](https://github.com/alist-org/alist/assets/56105412/9448c93f-e4ee-422e-ba6e-cd43cd8d7a8a)

**Reply by 455511dcvv** [reply]
aliyun JS-SDK V1.1.5
index.b4298eea.js:1 Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "application/x-js". Strict MIME type checking is enforced for module scripts per HTML spec.

是这个错误您看看
